app/globals.css

```
@@ -132,9 +147,182 @@
   }
   body {
     @apply bg-background text-foreground;
+    font-family: Tahoma, Verdana, Arial, sans-serif !important;
```

Stylelint (17.5.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)